src/components/MapView/Boundaries/index.tsx

```diff
@@ -26,6 +26,11 @@ function getAdminData(evt: any) {
   );
 }
 
+function onToggleHover(cursor: string, targetMap: MapboxGL.Map) {
+  // eslint-disable-next-line
```

Comment:
This is very rarely what you want to do (a blanket eslint disable. At most you generally want to disable a specific rule, if there's a good reason for it).

My guess is that in this case you're running into `no-param-reassign` or one of the `fp` rules? Which rule is this for?

Reply:
both ;)